content/BatchRequestContent.md

```diff
@@ -21,7 +22,34 @@ A batch cannot hold more that 20 requests and requests cannot contain a batch re
 
 Requests that require additional consent, not already obtained, should not be included in a batch.
 
+## Usage Examples
+
+```cs
+// create the batch request
+var batchRequestContent = new BatchRequestContent(graphServiceClient);
+var requestInformation = graphServiceClient.Users.ToGetRequestInformation();
+
+// add step using requestInformation
+var requestStepId = await batchRequestContent.AddBatchRequestStepAsync(requestInformation);
+
+// add step using native platform HTTP request
+var httpRequestMessage = new HttpRequestMessage(HttpMethod.Get, "http://example.com");
+var secondRequestStepId = batchRequestContent.AddBatchRequestStep(httpRequestMessage);
+
+// send and get back response
+var batchResponseContent = await graphServiceClient.Batch.PostAsync(batchRequestContent);
+
+// enumerate list of failed requests and create a new batch request
+var statusCodes = await batchResponseContent.GetResponsesStatusCodesAsync();
+if(statusCodes.Any())
+{
+    var retryBatch = batchRequestContent.NewBatchWithFailedRequests(rateLimitedResponses);
+    // send and get back response
+    var retryBatchResponseContent = await graphServiceClient.Batch.PostAsync(retryBatch);
+}
+```
+
 ## References
 
-- [1] [Graph Documentation]( https://developer.microsoft.com/en-us/graph/docs/concepts/json_batching)
-- [2] [OData](https://www.oasis-open.org/committees/download.php/60365/odata-json-format-v4.01-wd02-2017-03-24.docx)
+1. [Graph Documentation]( https://learn.microsoft.com/graph/json-batching)
+1. [OData](https://www.oasis-open.org/committees/download.php/60365/odata-json-format-v4.01-wd02-2017-03-24.docx)
```

Comment:
```suggestion
2. [OData](https://www.oasis-open.org/committees/download.php/60365/odata-json-format-v4.01-wd02-2017-03-24.docx)
```

Reply:
This is intentional. The numbers will autoincrement in markdown. https://stackoverflow.com/questions/12145232/create-an-automatically-numbered-list-in-markdown-format